DOCKER.md

```diff
@@ -81,15 +72,15 @@ $ docker-compose exec database psql -h database -Upostgres circulate_development
 
 #### Rails console
 ```
-$ docker-compose exec web rails c
+$ docker-compose run web rails c
```

Comment:
I suspect that changing these from `exec` to `run` will result in conflicts when trying to run two or more things based on the `web` service at the same time. I'll give it a try.

Reply:
This console command is OK with `web` already running, so I'll resolve this comment. But see later for the tests ...